evap/staff/tests/test_tools.py

```diff
@@ -65,16 +67,24 @@ def setUpTestData(cls):
             represented_users=[cls.user1],
             cc_users=[],
             ccing_users=[cls.user1, cls.user2],
-            is_superuser=True
+            is_superuser=True,
         )
         cls.course1 = baker.make(Course, responsibles=[cls.main_user])
         cls.course2 = baker.make(Course, responsibles=[cls.main_user])
         cls.course3 = baker.make(Course, responsibles=[cls.other_user])
-        cls.evaluation1 = baker.make(Evaluation, course=cls.course1, name_de="evaluation1", participants=[cls.main_user, cls.other_user])  # this should make the merge fail
-        cls.evaluation2 = baker.make(Evaluation, course=cls.course2, name_de="evaluation2", participants=[cls.main_user], voters=[cls.main_user])
-        cls.evaluation3 = baker.make(Evaluation, course=cls.course3, name_de="evaluation3", participants=[cls.other_user], voters=[cls.other_user])
+        cls.evaluation1 = baker.make(
+            Evaluation, course=cls.course1, name_de="evaluation1", participants=[cls.main_user, cls.other_user]
+        )  # this should make the merge fail
+        cls.evaluation2 = baker.make(
+            Evaluation, course=cls.course2, name_de="evaluation2", participants=[cls.main_user], voters=[cls.main_user]
+        )
+        cls.evaluation3 = baker.make(
+            Evaluation, course=cls.course3, name_de="evaluation3", participants=[cls.other_user], voters=[cls.other_user]
+        )
         cls.contribution1 = baker.make(Contribution, contributor=cls.main_user, evaluation=cls.evaluation1)
-        cls.contribution2 = baker.make(Contribution, contributor=cls.other_user, evaluation=cls.evaluation1)  # this should make the merge fail
+        cls.contribution2 = baker.make(
+            Contribution, contributor=cls.other_user, evaluation=cls.evaluation1
+        )  # this should make the merge fail
```

Comment:
actually I kinda got used to this already. what do you think?

Reply:
Hmm I think this one is okay. However, for some of the other, weird line breaks inside a subscription (`[]`), I think it would be better to move the comment to nudge the formatting.